Autopilot/PathManager/Src/landingTakeoffManager.cpp

```diff
@@ -0,0 +1,184 @@
+#include "waypointManager.hpp"
+#include "landingTakeoffManager.hpp"
+#include <math.h>
+#include "vectorClass.hpp"
+
+double LandingManager::changingAltitude(_WaypointManager_Data_In input, _PathData aimingPoint, _PathData intersectionPoint, _PathData stoppingPoint)
+{
+    //vector declaration
+    Vector3D aPoint(aimingPoint.longitude, aimingPoint.latitude, aimingPoint.altitude); //aiming point
+    Vector3D iPoint(intersectionPoint.longitude, intersectionPoint.latitude, intersectionPoint.altitude); //intersection point
+    Vector3D sPoint(stoppingPoint.longitude, stoppingPoint.latitude, stoppingPoint.altitude); //stopping point
+    Vector3D cPoint(input.longitude, input.latitude, input.altitude); //current point
+    Vector3D projectedPoint;
+
+    //calculating the vectors
+    Vector3D vectorAI(iPoint.x - aPoint.x, iPoint.y - aPoint.y, iPoint.z - aPoint.z);
+    Vector3D vectorAS(sPoint.x - aPoint.x, sPoint.y - aPoint.y, sPoint.z - aPoint.z);
+    Vector3D vectorAC(cPoint.x - aPoint.x, cPoint.y - aPoint.y, cPoint.z - aPoint.z);
+    Vector3D normal;
```

Comment:
the variable `normal` is blue here... that kinda worries me because git is interpreting it as something. Maybe change the name to normalVector?

Reply:
had that issue before, it doesn't actually do anything but sure, I'll change it to normalVector.